introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is a full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive 3-clause BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
```

Comment:
This sentence sounds more like an advertisement.  I would get rid of it or move it later in the paper. This isn't supposed to be a paper on the community, per se.  And @certik is an author of the paper so it is weird to mention his name here.